src/index.tsx

```diff
@@ -1,6 +1,36 @@
 import { createRoot } from 'react-dom/client';
+import {
+  Navigate,
+  HashRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+
 import { App } from './App';
 
-createRoot(document.getElementById('root') as HTMLDivElement).render(<App />);
+createRoot(document.getElementById('root') as HTMLDivElement).render(
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<h1>HomePage</h1>} />
+        <Route path="home" element={<Navigate to={'/'} />} />
+
+        <Route path="phones" element={<h1>Phones</h1>}>
+          <Route path=":itemId?" element={<h1>PhonesItems</h1>} />
```

Comment:
You can remove ? and  change itemId to phoneId. 
Same with tablets and accessories.